WooCommerce/Classes/ViewRelated/Products/Variations/Add Attributes/AddAttributeOptionsViewModel.swift

```diff
@@ -196,6 +211,14 @@ private extension AddAttributeOptionsViewModel {
         state.isSyncing = true
         stores.dispatch(fetchOptions)
     }
+
+    /// Returns a filtered version of `state.optionsAdded` where options that exists in `state.optionsOffered` are removed.
+    ///
+    func filterOptionsAdded() -> [ProductAttributeTerm] {
```

Comment:
nit: "OptionsAdded" sounds like a data set for "options selected by users" (offered section). Maybe rename it according to section title - `filterOptionsExisting`?